cartography/intel/aws/rds.py

```diff
@@ -123,6 +123,192 @@ def sync_rds_reserved_db_instances(
     cleanup_rds_reserved_db_instances(neo4j_session, common_job_parameters)
 
 
+@timeit
+@aws_handle_regions
+def get_rds_security_groups(boto3_session: boto3.session.Session, region: str) -> List[Any]:
+    client = boto3_session.client('rds', region_name=region)
+    paginator = client.get_paginator('describe_db_security_groups')
+    secgroups: List[Any] = []
+    for page in paginator.paginate():
+        secgroups.extend(page['DBSecurityGroups'])
+    for secgroup in secgroups:
+        secgroup['region'] = region
+    return secgroups
+
+
+def load_rds_security_groups(session: neo4j.Session, secgroups: List[Dict], current_aws_account_id: str, aws_update_tag: int) -> None:
+    session.write_transaction(_load_rds_security_groups_tx, secgroups, current_aws_account_id, aws_update_tag)
+
+
+@timeit
+def _load_rds_security_groups_tx(
+    tx: neo4j.Transaction, data: List[Dict], current_aws_account_id: str,
+    aws_update_tag: int,
+) -> None:
+    ingest_rds_secgroup = """
+    UNWIND {secgroups} as rds_secgroup
+        MERGE (secgroup:RDSSecurityGroup{id: rds_secgroup.DBSecurityGroupArn})
```

Comment:
We should build a relationship between RDSSecurityGroup and SecurityGroup object. SecurityGroup object is used to link with RDS and that's the RDSSecurityGroup.